Build failure: python313Packages.rapidfuzz

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

1. nix shell nixpkgs#python313Packages.rapidfuzz

### Can Hydra reproduce this build failure?

Hydra is currently rebuilding this package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing python-runtime-deps-check-hook
Using pythonRuntimeDepsCheckHook
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/zcxmjzqg1fy2cvaj5k3hpsz938yp0d7d-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file "source/tools/test_submodules.py"
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
applying patch /nix/store/00k8ksfy5bp2lj2mazd8v0h51c68xy8l-support-taskflow-3.11.0.patch
patching file CMakeLists.txt
applying patch /nix/store/jf6j5ww4kv80fh38g0vsfdyc38lviq92-support-taskflow-4.0.0.patch
patching file CMakeLists.txt
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
no configure script, doing nothing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
[1m* Getting build dependencies for wheel...[0m
[1m* Building wheel...[0m
[32m*** [1mscikit-build-core 0.11.5[0m using [34mCMake 4.1.2[39m[0m [31m(wheel)[0m
[32m***[0m [1mConfiguring CMake...[0m
loading initial cache file /nix/var/nix/builds/nix-28441-4110120110/tmphur4k3b2/build/CMakeInit.txt
-- The C compiler identification is Clang 21.1.8
-- The CXX compiler identification is Clang 21.1.8
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/z0dmrxbgqi98f7n8ljza91y5qazdyhcp-clang-wrapper-21.1.8/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/z0dmrxbgqi98f7n8ljza91y5qazdyhcp-clang-wrapper-21.1.8/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- CMAKE_AR="/nix/store/z0dmrxbgqi98f7n8ljza91y5qazdyhcp-clang-wrapper-21.1.8/bin/ar" ; CMAKE_C_COMPILER_AR="CMAKE_C_COMPILER_AR-NOTFOUND" ; CMAKE_CXX_COMPILER_AR="CMAKE_CXX_COMPILER_AR-NOTFOUND"
-- CMAKE_RANLIB="/nix/store/z0dmrxbgqi98f7n8ljza91y5qazdyhcp-clang-wrapper-21.1.8/bin/ranlib" ; CMAKE_C_COMPILER_RANLIB="CMAKE_C_COMPILER_RANLIB-NOTFOUND" ; CMAKE_CXX_COMPILER_RANLIB="CMAKE_CXX_COMPILER_RANLIB-NOTFOUND"
-- Found Python: /nix/store/qzgxkh5rnlswnj5ywhag0djwvn83kr3r-python3-3.13.11/bin/python3.13 (found version "3.13.11") found components: Interpreter Development.Module
-- Check CPU architecture is x64
-- Check CPU architecture is x64 - no
-- Check CPU architecture is x86
-- Check CPU architecture is x86 - no
-- Taskflow found. Headers: /nix/store/02b78v62sj9q614qxhafjf1ab2njzvpx-taskflow-4.0.0/include/
-- Using system supplied version of Taskflow
-- Using system supplied version of rapidfuzz-cpp
-- Performing Test HAVE_CXX_ATOMICS_INT_WITHOUT_LIB
-- Performing Test HAVE_CXX_ATOMICS_INT_WITHOUT_LIB - Failed
-- Performing Test HAVE_CXX_ATOMICS_SIZE_T_WITHOUT_LIB
-- Performing Test HAVE_CXX_ATOMICS_SIZE_T_WITHOUT_LIB - Failed
-- Performing Test HAVE_CXX_ATOMICS_VOID_PTR_WITHOUT_LIB
-- Performing Test HAVE_CXX_ATOMICS_VOID_PTR_WITHOUT_LIB - Failed
-- Performing Test HAVE_CXX_ATOMICS_UNSIGNED_WITHOUT_LIB
-- Performing Test HAVE_CXX_ATOMICS_UNSIGNED_WITHOUT_LIB - Failed
-- Performing Test HAVE_CXX_ATOMICS_UINT64_WITHOUT_LIB
-- Performing Test HAVE_CXX_ATOMICS_UINT64_WITHOUT_LIB - Failed
-- Performing Test HAVE_CXX_ATOMICS_INT_WITH_LIB
-- Performing Test HAVE_CXX_ATOMICS_INT_WITH_LIB - Failed
-- Performing Test HAVE_CXX_ATOMICS_SIZE_T_WITH_LIB
-- Performing Test HAVE_CXX_ATOMICS_SIZE_T_WITH_LIB - Failed
-- Performing Test HAVE_CXX_ATOMICS_VOID_PTR_WITH_LIB
-- Performing Test HAVE_CXX_ATOMICS_VOID_PTR_WITH_LIB - Failed
-- Performing Test HAVE_CXX_ATOMICS_UNSIGNED_WITH_LIB
-- Performing Test HAVE_CXX_ATOMICS_UNSIGNED_WITH_LIB - Failed
-- Performing Test HAVE_CXX_ATOMICS_UINT64_WITH_LIB
-- Performing Test HAVE_CXX_ATOMICS_UINT64_WITH_LIB - Failed
[31mCMake Error at src/rapidfuzz/CMakeLists.txt:171 (message):
  No native support for std::atomic<int>, or libatomic not found!

[0m
-- Configuring incomplete, errors occurred!
[31m
[1m***[0m [31mCMake configuration failed[0m
[31mFailed to build C++ Extension in a packaged build[0m

[91mERROR[0m Backend subprocess exited when trying to invoke build_wheel
```

### Additional context

_No response_

### System metadata

 - system: `"aarch64-darwin"`
 - host os: `Darwin 25.2.0, macOS 26.2`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.31.3`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/store/fbw0wnbshbly0dssrblywq9l6dlmp58k-nixpkgs/nixpkgs`

### Notify maintainers

@dotlambda

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+label%3A%220.kind%3A+build+failure%22).
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/rapidfuzz/default.nix
+++ b/pkgs/development/python-modules/rapidfuzz/default.nix
@@ -4,6 +4,7 @@
   buildPythonPackage,
   fetchFromGitHub,
   fetchpatch,
+  clang-tools,
   cmake,
   cython,
   ninja,
@@ -53,6 +54,9 @@ buildPythonPackage rec {
     cython
     ninja
     scikit-build-core
+  ]
+  ++ lib.optionals stdenv.cc.isClang [
+    clang-tools # provides wrapped clang-scan-deps
   ];
 
   dontUseCmakeConfigure = true;

